app/server/api/auth_api.py

```diff
@@ -76,7 +76,7 @@ def post(self):
         email_ok = False
 
         whitelisted_emails = EmailWhitelist.query\
-            .filter_by(email=email, referral_code=referral_code, used=False) \
```

Comment:
This was causing partial email matchers (@gmail.com) to not show in the list

Reply:
nice catch